ironfish/src/wallet/wallet.test.ts

```diff
@@ -1343,28 +1343,6 @@ describe('Accounts', () => {
   })
 
   describe('addPendingTransaction', () => {
-    it('should add transactions to accounts involved in the transaction', async () => {
-      const { node } = await nodeTest.createSetup()
-
-      const accountA = await useAccountFixture(node.wallet, 'a')
-      const accountB = await useAccountFixture(node.wallet, 'b')
-
-      const blockA1 = await useMinerBlockFixture(node.chain, undefined, accountA, node.wallet)
-      await expect(node.chain).toAddBlock(blockA1)
-      await node.wallet.updateHead()
-
-      const addSpyA = jest.spyOn(accountA, 'addPendingTransaction')
-      const addSpyB = jest.spyOn(accountB, 'addPendingTransaction')
-
-      await useTxFixture(node.wallet, accountA, accountA)
-
-      // tx added to accountA
-      expect(addSpyA).toHaveBeenCalledTimes(1)
-
-      // tx not added to accountB
-      expect(addSpyB).toHaveBeenCalledTimes(0)
-    })
-
```

Comment:
I was sketched out we were removing a test, but I see this is mirrored/replaced over in `account.test.ts`